scripts/generate_dag.py

```diff
@@ -221,7 +226,60 @@ def print_airflow_variables(dataset_id: str, dag_contents: str, env: str):
         elif var.startswith("value."):
             var = var.replace("value.", "", 1)
         print(f"  - {var}")
-    print()
+        missing_var = var.split(".")[1]
+        user_input = input(f"Airflow Variable Missing | {missing_var} value: ")
+        valid_input = validate_user_input(missing_var, user_input)
+        var_json[dataset_id][missing_var] = valid_input
+
+    print(f"\n{dataset_id} Variables file:")
+    print(f"\n{json.dumps(var_json, indent = 4)}")
```

Comment:
Double check the Google JS Style Guide for indenting objects. I think it's 2 spaces.